pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -140,60 +140,80 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 		c.AddTask(elb)
 	}
 
-	// Create security group for API ELB
-	{
-		t := &awstasks.SecurityGroup{
-			Name:      s(b.ELBSecurityGroupName("api")),
-			Lifecycle: b.Lifecycle,
-
-			VPC:              b.LinkToVPC(),
-			Description:      s("Security group for api ELB"),
-			RemoveExtraRules: []string{"port=443"},
+	createSecurityGroup := true
+	if b.Cluster.Spec.SecurityGroups != nil {
+		if b.Cluster.Spec.SecurityGroups.ApiELB != nil {
+			glog.V(8).Infof("re-using security group: %s for api elb", fi.StringValue(b.Cluster.Spec.SecurityGroups.ApiELB))
```

Comment:
You just set the ID field on the task; it'll then pick up the existing security group: https://github.com/kubernetes/kops/blob/master/upup/pkg/fi/cloudup/awstasks/securitygroup.go#L108-L116

Reply:
Need more context, because in my mind I am doing that :)